packages/extensions/openapi-to-cadl/test/analyzeText/tsp-output/package.json

```diff
@@ -4,13 +4,13 @@
   "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms. The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview</a>.",
   "license": "MIT",
   "dependencies": {
-    "@typespec/compiler": "^0.41.0",
```

Comment:
We dont need to output a package.json anymore since the specs repo relies on a global package.json now

Reply:
We dont need to include this change in this PR it's just a heads up so we remove it eventually

Reply:
Makes sense I have added this change, you can still generate with --include-package, but the default is false.